src/rateEngine/LoadProfile.ts

```diff
@@ -97,6 +99,20 @@ class LoadProfile {
     });
   }
 
+  maxPerDayByMonth(): Array<number> {
+    const months = times(12, (i) => i);
+    const expanded = this.expanded();
+
+    return months.map((m) => {
+      const monthLoads = expanded.filter(({ month }) => m === month);
+
+      // chunk monthly loads by day (31-element array for January, etc.)
+      const dayLoads = Object.values(groupBy(monthLoads, (val) => val.date));
```

Comment:
This was an interesting problem. I originally was manually chunking every month into 24-element arrays but forgot that DST adds some complexity. This solution partitions the month by each reading's `date` field, which is definitely safer

Reply:
Good eye!